opm/simulators/flow/FlowProblem.hpp

```diff
@@ -1694,6 +1694,11 @@ class FlowProblem : public GetPropType<TypeTag, Properties::BaseProblem>
     PffGridVector<GridView, Stencil, PffDofData_, DofMapper> pffDofData_;
     TracerModel tracerModel_;
 
+
+
+protected:
+    ActionHandler<Scalar> actionHandler_;
```

Comment:
if we are to introduce this member here, don't we need to move the initialization from the Blackoil child class also? and what happens with compositional, which afaik was the reason it was put in the Blackoil class?